app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level
+
+    def player_info(self) -> None:

```

Comment:
Invalid type hint for return value. It returns `str`
```suggestion
    def player_info(self) -> str:

```